engines/libblkio.c

```diff
@@ -287,12 +287,12 @@ static int total_threaded_subjobs(bool hipri)
 	unsigned int i;
```

Comment:
You forgot to kill 'i'.

Reply:
> You forgot to kill 'i'.

I was trying to get libblkio to build on my system to compile and test this change but it keeps hanging during the ``meson compile -C build`, so I'm having to push without being able to compile. Just removed `td` and `i`.